test/functional/services/remote/leadfoot_command.js

```diff
@@ -8,18 +8,25 @@ const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 
 let attemptCounter = 0;
-async function attemptToCreateCommand(log, server, chromedriverApi) {
+async function attemptToCreateCommand(log, server, driverApi) {
   const attemptId = ++attemptCounter;
+  let capabilities = {};
+
+  switch(driverApi.getBrowserName()) {
+    default:
+      capabilities = { browserName: 'firefox' };
+  }
 
   log.debug('[leadfoot:command] Creating session');
-  const session = await server.createSession({ browserName: 'chrome' });
+  const session = await server.createSession({}, capabilities);
   if (attemptId !== attemptCounter) return; // abort
 
   log.debug('[leadfoot:command] Registerying session for teardown');
-  chromedriverApi.beforeStop(async () => session.quit());
+  driverApi.beforeStop(async () => session.quit());
   if (attemptId !== attemptCounter) return; // abort
 
   log.debug('[leadfoot:command] Completing session capabilities');
+  log.debug(session);
```

Comment:
We probably shouldn't log the session object all the time, I don't think it's terribly useful, but maybe I'm wrong

Reply:
Nah, I agree. Removed.